packages/@lwc/integration-karma/helpers/test-utils.js

```diff
@@ -577,19 +576,29 @@ window.TestUtils = (function (lwc, jasmine, beforeAll) {
                     expect(calls).toHaveSize(matchers.length);
                     for (let i = 0; i < matchers.length; i++) {
                         const matcher = matchers[i];
-                        const call = calls[i][0];
-                        const message = typeof call === 'string' ? call : call.message;
-                        if (typeof matcher === 'string') {
-                            expect(message).toContain(matcher);
-                        } else {
-                            expect(message).toMatch(matcher);
-                        }
+                        const args = calls[i];
+                        const argsString = args.map((arg) => stringifyArg(arg)).join(' ');
+                        expect(argsString).toMatch(matcher);
```

Comment:
Looks like I missed that [`toMatch`](https://jasmine.github.io/api/edge/matchers.html#toMatch) on a strig does the same thing as `toContain`. Good find!